sql/moz-fx-data-shared-prod/telemetry_derived/attributed_windows_desktop_funnel_new_profiles_v1/query.sql

```diff
@@ -0,0 +1,92 @@
+CREATE TEMP FUNCTION bucket_country(country STRING) AS (
+  CASE
+  WHEN
+    country IN ('US', 'CA', 'DE', 'FR', 'GB')
+  THEN
+    [country, 'tier-1']
+  WHEN
+    country IN ('MX', 'BR')
+  THEN
+    [country, 'non-tier-1']
+  ELSE
+    ['non-tier-1']
+  END
+);
+
+WITH pop AS (
+  SELECT
+    DISTINCT client_id,
+    normalized_country_code AS country_code,
+    DATE(submission_timestamp) AS date
+  FROM
+    telemetry.new_profile
+  WHERE
+    DATE(submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 8 DAY)
+    AND @submission_date
+    AND normalized_channel = 'release'
+    AND DATE_DIFF(  -- Only use builds from the last month
+      DATE(submission_timestamp),
+      SAFE.PARSE_DATE('%Y%m%d', SUBSTR(application.build_id, 0, 8)),
+      MONTH
+    ) <= 1
+    AND payload.processes.parent.scalars.startup_profile_selection_reason = 'firstrun-created-default'
+    AND environment.system.os.name = 'Windows_NT'
+     -- TODO: make sure source is reliably null and not 0
+    AND environment.settings.attribution.source IS NOT NULL
+    AND environment.partner.distribution_id IS NULL
+    AND coalesce(environment.settings.attribution.ua, '') != 'firefox'
+),
+raw_info AS (
+  SELECT
+    client_id,
+    submission_date,
+    country AS country_code,
+    any_value(days_seen_bits) AS days_seen_bits,
+  FROM
+    telemetry.clients_last_seen
+  WHERE
+    submission_date
+    BETWEEN @submission_date
+    AND DATE_ADD(@submission_date, INTERVAL 8 DAY)
```

Comment:
For this to work, this query should only be processed 8 days after the date. Hopefully this is possible with airflow?

Reply:
I split these in two.